keras/backend/tensorflow_backend.py

```diff
@@ -1307,6 +1339,22 @@ def clip(x, min_value, max_value):
     return tf.clip_by_value(x, min_value, max_value)
 
 
+def clipnorm(x, max_value):
```

Comment:
I don't think this one belongs. In any case, the name is questionable (inconsistent with TF), it does not have a Theano implementation, and no direct TF equivalent. Please remove.